src/test/resources/test.properties

```diff
@@ -4,6 +4,15 @@ spring.profiles.active=test
 # Liquibase Database Migration
 liquibase.contexts=test
 
+#######################################################################################################################
+# Application
+#######################################################################################################################
+app.name=Voyage
```

Comment:
These props are getting duplicated from application.properties - by default these should come from application, unless the user wants to override them for tests